views/layout.html

```diff
@@ -291,15 +298,19 @@
         </div>
     </div>
     </main>
-    <footer class="page-footer" style="background-color: #536dfe;">
-          <div class="footer-copyright">
-            <div class="container">
+    <footer class="footer">
+        <div class="container">
             <div class="left"> © 2015-2016 StopStalk </div>
             <div class="right">
-               <span>Carefully crafted by: <a href="http://www.raj454raj.com" style="color: white;" target="_blank">Raj Patel</a></span>
+                <span><a href="/stopstalk/default/faq" style="color: white;" target="_blank">FAQs</a></span>
+                <span> | </span>
+                <span><a href="/stopstalk/default/contact_us" style="color: white;" target="_blank">Contact Us</a></span>
```

Comment:
Also since you are adding one more commit, add one more item - 
`Contributors` with url https://github.com/stopstalk/stopstalk-deployment/graphs/contributors